7.2.0_2.4/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM buildpack-deps:jessie
+
+MAINTAINER Daron <daron@expert360.com>
```

Comment:
This directive has been deprecated, please remove it.

Reply:
done